docs/tasks/debug-application-cluster/audit.md

```diff
@@ -70,3 +63,228 @@ the file name (before the file extension) and create a new audit log file.
 Kubernetes may delete old log files when creating a new log file; you can configure
 how many files are retained and how old they can be by specifying the `audit-log-maxbackup`
 and `audit-log-maxage` options.
+
+## Advanced audit
+
+Kubernetes 1.7 expands auditing with experimental functionality such as event
+filtering and a webhook for integration with external systems. The rest of this
+document covers features that are __alpha__ and may change in backward incompatible
+ways.
+
+Enable the alpha auditing features using the `AdvancedAuditing` feature gate on
+the [kube-apiserver][kube-apiserver]:
+
+```
+--feature-gates=AdvancedAuditing=true
+```
+
+`AdvancedAuditing` is composed of two phases. Policy, which determines what's recorded,
```

Comment:
nit: "phases" is confusing here. How about "AdvancedAuditing is customizeable in two ways."?